### PR TITLE
Using nodeSelector & Tolerations to schedule windows test pods

### PR DESCRIPTION
**Pull request by YangLu1031**
1. For simplicity, we taint linux node and untaint windows node to schedule pause image on windows node for pod startup latency test before. But now need to schedule prometheus server pods on linux node as well, so add nodeSelector & Tolerations to schedule pause image on windows node. 
2. Bumped to latest pause image version.

### CODE REVIEW

File: clusterloader2/testing/node-throughput/rc.yaml [hunk 1]
```diff
@@ -24,6 +28,12 @@ spec:
       # Add not-ready/unreachable tolerations for 15 minutes so that node
       # failure doesn't trigger pod deletion.
       tolerations:
+      {{if .WindowsNodeTest}}
+      - key: "node.kubernetes.io/os"
+        operator: "Equal"
+        value: "win1809"
```
Comment (pjh) [inline clusterloader2/testing/node-throughput/rc.yaml]: 1809 will be going away soon, we should switch to 2019 instead. Could be in this PR or a follow-up.

Comment (YangLu1031) [reply clusterloader2/testing/node-throughput/rc.yaml]: I see, seems need to change to more relax operator? like  "Exists"

**Pull request merged**